glib/src/subclass/signal.rs

```diff
@@ -361,8 +385,8 @@ impl Signal {
     /// Create a new builder for a signal.
     pub fn builder<'a>(
         name: &'a str,
-        param_types: &'a [Type],
-        return_type: Type,
+        param_types: &'a [(Type, bool)],
+        return_type: (Type, bool),
```

Comment:
I'm not very happy about this, as the user can't tell it those booleans are for setting the param/return type as static scope. Any ideas?

Reply:
Maybe there could be some kind of new type `SignalType(ffi::GType)` that is `From<Type> for SignalType` (and the inverse of course) and also has a `fn with_static_scope(t: Type) -> Self`, and we'd work with those here instead?

Reply:
Ah, great. That can work and simplify the SignalQuery part as well. I like it.